test/models/inventory.ts

```diff
@@ -30,8 +30,8 @@ export class Inventory {
   @prop({ required: true, refPath: 'refItemPathName' })
   public kind!: Ref<Beverage | Scooter>;
 
-  @arrayProp({ required: true, refPath: 'refItemPathName' })
-  public irp!: Ref<Beverage | Scooter>[];
+  @arrayProp({ refPath: 'refItemPathName' })
```

Comment:
why did `required` get removed here?

Reply:
One of the tests failed because it didn't specify the `irp` in the constructor. You can test it yourself by reverting this part. I thought it was an oversight, but if it wasn't, let me know how to fix it and I'll gladly do so.

Reply:
changed it back, no errors for me

Reply:
I have an error in `shouldAdd.test.ts` at lines 133 and 140. Maybe we have different configurations?

Reply:
Yep, the most recent build failed because of `shouldAdd.test.ts`.

Reply:
noticed my tests passed while script `buildtests` gave errors
-> and i noticed that the array-create's are first, i turned that around otherwise array-create's would be default and suggest wrong types
--> but now there is another problem, and i know why

Reply:
should be fixed in 88b10bff656bb8ad2e5119bb495cbf319c98a8b8